src/connectors/autohide/autohide.hoc.js

```diff
@@ -1,44 +1,37 @@
-import React, { Component } from 'react'
+import React, { useState, useRef } from 'react'
 
-export const withAutoHider = WrappedComponent => {
-  class AutoHiderHOC extends Component {
-    initialDelay = 5000
-    delay = 2500
-    state = {
-      hasTimedOut: false
-    }
+export const withAutoHider = (WrappedComponent) => ({ ...props }) => {
```

Comment:
Do we still need this since we talked about the doorhanger not auto-dismissing? Happy to hold off on deleting it for another PR, just wanted to make sure.

Reply:
Nope, we can totally delete this!

Reply:
There's still some code cleanup left to do with removing un-needed functions. It'll be much easier now that we've ditched most the code in the `v4` branch.